tests/test_variable.py

```diff
@@ -5,6 +5,60 @@
 
 import relentless
 
+class test_Variable(unittest.TestCase):
+    """Unit tests for relentless.variable.Variable."""
+
+    def test_operations(self):
+        """Test arithmetic operations on variables."""
+        #create dependent variables
+        v = relentless.variable.IndependentVariable(value=2.0)
+        w = relentless.variable.IndependentVariable(value=4.0)
+
+        #addition
+        x = v+w
+        self.assertAlmostEqual(x.value, 6.0)
+        x = v+2.0
+        self.assertAlmostEqual(x.value, 4.0)
+        x = 2.0+w
+        self.assertAlmostEqual(x.value, 6.0)
+
+        #subtraction
+        x = v-w
+        self.assertAlmostEqual(x.value, -2.0)
+        x = v-1.0
+        self.assertAlmostEqual(x.value, 1.0)
+        x = 1.0-w
+        self.assertAlmostEqual(x.value, -3.0)
+
+        #multiplication
+        x = v*w
+        self.assertAlmostEqual(x.value, 8.0)
+        x = v*1.5
+        self.assertAlmostEqual(x.value, 3.0)
+        x = 1.5*w
+        self.assertAlmostEqual(x.value, 6.0)
+
+        #division
+        x = w/v
+        self.assertAlmostEqual(x.value, 2.0)
+        x = v/2.0
+        self.assertAlmostEqual(x.value, 1.0)
+        x = 3.0/w
+        self.assertAlmostEqual(x.value, 0.75)
+
+        #exponentiation
+        x = w**v
+        self.assertAlmostEqual(x.value, 16.0)
+        x = v**3.0
+        self.assertAlmostEqual(x.value, 8.0)
+
+        #negation
+        x = -w
+        self.assertAlmostEqual(x.value, -4.0)
+
+        #string conversion
+        self.assertEqual(str(x), str(x.value))
```

Comment:
Small thing, make this check the expected string:
```suggestion
        self.assertEqual(str(x), "4")
```
(or however this formats)